### PR TITLE
Added jokes and latency checker.

### PR DESCRIPTION
**Pull request by harshhh-dev**
Jokes - gets a random joke from discord-jokes.

Latency - measures response times.

### CODE REVIEW

**Comment by harshhh-dev** [comment]
Removed jokes since you don't like to rely on random libs

File: src/commands/oftenAsked/google.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { create_command } from '../../command';
```
Comment (lenscas) [inline src/commands/oftenAsked/google.ts:1]: Why are these changes here again?

File: src/commands/oftenAsked/ping.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { create_command } from '../../command';
+
+export const command = create_command(
```
Comment (lenscas) [inline src/commands/oftenAsked/ping.ts:3]: use https://github.com/lenscas/thanks_bot_2/blob/ed83f073d020f7d663c70fc60f1fe8d511c457ff/src/command.ts#L47 instead so it is limited to the command channel, rather than being available everywhere.

It also should go in the `bot` group. As it is related to the bot, rather than giving an answer to an often asked question

**Review by lenscas — CHANGES_REQUESTED** [review]

File: src/commands/oftenAsked/ping.ts (hunk 2 above)
Comment (harshhh-dev) [reply src/commands/oftenAsked/ping.ts:3]: Oh okay. Will fix :+1:

File: src/commands/oftenAsked/google.ts (hunk 1 above)
Comment (harshhh-dev) [reply src/commands/oftenAsked/google.ts:1]: Frogot to update fork.
